vernac/vernacentries.mli

```diff
@@ -87,6 +88,9 @@ type t = {
   clearbody: bool option;
 }
 
+module Observer : Summary.OBSERVABLE
+  with type value = (Declare.Hook.S.t -> unit) list Attributes.attribute
```

Comment:
this could equivalently be `hook list` (current version), `hook option` or just `hook` since closures to unit already form a monoid. IDK which is nicer to work with.

Reply:
or `Declare.Hook.t list` (in which case I guess we do want the list since composing is more opaque)